modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/jaxrs/exception/mapper/NoSuchModelExceptionMapper.java

```diff
@@ -15,52 +15,25 @@
 package com.liferay.portal.vulcan.internal.jaxrs.exception.mapper;
 
 import com.liferay.portal.kernel.exception.NoSuchModelException;
-import com.liferay.portal.kernel.servlet.HttpMethods;
 import com.liferay.portal.vulcan.jaxrs.exception.mapper.BaseExceptionMapper;
 import com.liferay.portal.vulcan.jaxrs.exception.mapper.Problem;
 
-import javax.servlet.http.HttpServletRequest;
-
-import javax.ws.rs.core.Context;
 import javax.ws.rs.core.Response;
 
 /**
  * Converts any {@code NoSuchModelException} to a {@code 404} error.
  *
- * <p>In the case of a DELETE request a {@code 204} (NO-CONTENT) response is
- * returned so the request is idempotent.
  *
  * @author Alejandro Hernández
  * @review
  */
 public class NoSuchModelExceptionMapper
 	extends BaseExceptionMapper<NoSuchModelException> {
 
-	@Override
-	public Response toResponse(NoSuchModelException noSuchModelException) {
-		String method = _httpServletRequest.getMethod();
-
-		if (method.equals(HttpMethods.DELETE)) {
-			return Response.status(
-				Response.Status.NO_CONTENT
-			).build();
-		}
-
-		return super.toResponse(noSuchModelException);
-	}
-
 	@Override
 	protected Problem getProblem(NoSuchModelException noSuchModelException) {
 		return new Problem(
 			Response.Status.NOT_FOUND, noSuchModelException.getMessage());
 	}
 
-	@Override
```

Comment:
Hi @sergiojimcos! Why do you remove this?

Reply:
If I remove this, when the exception is thrown it will appear in the log trace as WARN. If it is true, it won't be shown in the log. The point is, should we show the WARN message in the log or in the response of a request?

Reply:
+1. I think by removing it, you are changing the flow when an exception of this type is launched, for example by printing a log trace and sanitizing the exception

Reply:
Great, thanks guys, I will undo that part